src/Equinor.ProCoSys.IPO.Query/GetInvitationById/ParticipantDto.cs

```diff
@@ -44,7 +46,9 @@ public class ParticipantDto
         public bool Attended { get; }
         [Obsolete("Use isSigner")]
         public bool CanSign { get; }
+        // IsSigner can be false even if the participant is in theory a signer, if the IPO is cancelled.
```

Comment:
Is this solution chosen because of backward functionality? My first thought is that IsSigner should not be affacted by IPO cancelled or not ... the UI (and backend validator) should prevent signing of IPO cancelled

Reply:
Yes that was the thought, but I agree that with this wording we should just return if the participant IS signer, not if they CAN sign